src/DetailsView/components/scan-incomplete-warning.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { ScanIncompleteWarningId } from 'common/types/scan-incomplete-warnings';
+import { forOwn, isEmpty } from 'lodash';
+import { MessageBar, MessageBarType } from 'office-ui-fabric-react';
+import * as React from 'react';
+
+export type ScanIncompleteWarningProps = {
+    warnings: ScanIncompleteWarningId[];
+    onRender: () => void;
```

Comment:
What is this for? Presuming it's for the new onIframePermissionWarningShown, do we really want to be sending telemetry per-re-render? I'd expect it to be more appropriate to do that in response to scan complete actions with warnings present...

Reply:
Yes, I was thinking about this and removing it as well; this pattern also doesn't work for other warnings that might be shown.